test/services/PlaybookServiceTest.js

```diff
@@ -37,7 +37,7 @@ describe('Playbook Service', function () {
                 dateTimeComparisonValue: '19700101',
                 dataFormat: dateTimeDataFormatValue
```

Comment:
It would require a change on the gateway side; these are enums over there. Might be as simple as changing the DTO, but I definitely don't want to gate this pull on this.